kafka_consumer/check.py

```diff
@@ -181,9 +181,9 @@ def check(self, instance):
                 'consumer_group:%s' % consumer_group]
             self.gauge('kafka.consumer_offset', consumer_offset, tags=consumer_group_tags)
             if (topic, partition) not in highwater_offsets:
-                self.log.warn("Consumer offsets exist for topic: {topic} "
-                    "partition: {partition} but that topic partition doesn't "
-                    "actually exist in the cluster.".format(**locals()))
+                self.log.warn("Consumer offsets exist for topic: %s "
```

Comment:
It wasn't really a huge deal, particularly because `warning` is a pretty high log-level and most users will run at `info`... but thanks regardless :)